flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkRangeDistributionBucketing.java

```diff
@@ -108,6 +108,9 @@ public class TestFlinkIcebergSinkRangeDistributionBucketing {
   private static final PartitionSpec SPEC =
       PartitionSpec.builderFor(SCHEMA).hour("ts").bucket("uuid", NUM_BUCKETS).build();
   private static final RowType ROW_TYPE = FlinkSchemaUtil.convert(SCHEMA);
+  // FIXME (before merge): should it be parameterized test or hardcoded value with some comment that
```

Comment:
@pvary WDYT? Should I extract parameter tests or keep this hardcoded to not produce too much tests?